generate-bookmarks-html

```diff
@@ -21,6 +21,7 @@
 ################################################################################
 
 set -o errexit -o nounset -o pipefail
+shopt -s nullglob
```

Comment:
Good call. Calling it in a function and thinking it was scoped to that function was pretty silly (and useless).